frontend/src/components/login.js

```diff
@@ -1,10 +1,35 @@
 import React from 'react';
 
-const Login = () => (
-    <div>
+const Login = () => {
+    const handleSubmit = (evt) => {
+        evt.preventDefault();
+        console.log("Form submitted");
+    }
 
-    </div>
-);
+    return (
+            <div>
+            <form method="post" action="/st21-what-the-wine/public/index.html" onSubmit={handleSubmit}>
+            <fieldset>
+            <legend>Log In</legend>
+                <p>
+                    <label name="email">E-Mail Adress</label>
+                    <input type="email" name="email"></input>
+                </p>
+                <p>
+                    <label name="password">Create Password</label>
+                    <input type="password" name="password"></input>
+                </p>
+                <p>
+                <button type="submit">Log In</button>
+                <input type="reset" value="Reset the filter"></input>
+                </p>
+                
+            </fieldset>
+            </form>
+            </div>
+
+    )
+};
```

Comment:
I have 2 recommendations:
- Have you ever used [prettier](https://prettier.io/)? It's so handy and also the default tool in development nowadays. It formats your code instantly.
- About the formatting options itself, I'd recommend having a indentation of 2 spaces between lines (the tab size), I think it's 4 right now, 4 spaces will easily overflow the editor as the components get more complex. 2 spaces is the default value in `prettier` if you want to use it.

Reply:
Tried to look into prettier, need to figure out how that works. Thanks!